{"name": "新增用户", "status": "passed", "attachments": [{"name": "log", "source": "7f0b32f7-c7dd-456b-873c-e25bb6e846e6-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "inData", "value": "'{\"password\":\"\",\"password_confirm\": \"88888888\",\"username\":\"001\"}'"}, {"name": "exp", "value": "'{'code':500,'msg':\"Key:'userAddRequest.password'Error:Field validation for 'password' failed on the'required' tag\"}'"}], "start": 1604655474502, "stop": 1604655474533, "uuid": "16364c2e-4098-43ac-acb2-63b69e9d5ed7", "historyId": "f857598b92e6035d3ec25e0a063fad38", "testCaseId": "f56f10c6b6679a14e1ccd1e8f399d334", "fullName": "test_userManage.TestUserManage#test_add_user", "labels": [{"name": "suite", "value": "test_userManage"}, {"name": "subSuite", "value": "TestUserManage"}, {"name": "host", "value": "dadaxPC008"}, {"name": "thread", "value": "17104-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_userManage"}]}